notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>Ooo, you have users skip to the "Switch your kernel" section here, but if they bring a cluster, won't they need to fill in their values here? Or are these values only used in the creation process?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>If they already have a cluster, they don't have to fill this cell. These values only used in the creation process.</p>